tests/pythonSudo.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+# This shell script allows running the Python interpreter as root
+# NB: the user needs sudo NOPASSWD enabled
+sudo PYTHONPATH=$PYTHONPATH python3 "$@"
```

Comment:
This is needed in order to run tests that require root privileges from within an IDE, such as PyCharm.

Reply:
I'm uncomfortable turning on sudo with NOPASSWD on my personal machine. Running `sudo make test` should work though, right?

Reply:
Both `sudo ctest` and `sudo make test` work of course, when running the tests from command line. When running from within PyCharm though, there's no easy way to run with sudo, except for using this shell script as the interpreter. NOPASSWD will only be needed in this case, as PyCharm doesn't prompt for root credentials.